elytron/src/main/java/org/wildfly/extension/elytron/AuthenticationClientDefinitions.java

```diff
@@ -491,6 +494,18 @@ protected ValueSupplier<AuthenticationContext> getValueSupplier(ServiceBuilder<A
                 return () -> finalContext.apply(parentSupplier.get());
             }
 
+            @Override
+            protected void performRuntime(OperationContext context, ModelNode operation, ModelNode model) throws OperationFailedException {
+                commonRequirements(installService(context, model)).setInitialMode(ServiceController.Mode.ACTIVE).install();
```

Comment:
Does this service need to start eagerly?

Reply:
@pferraro You're right, I have updated it to be ON_DEMAND